docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,63 @@
+
+## Recommended package names
+
+The following is our recommendation for package names. A package maintainer may choose to diverge from it based on various reasons, such as, a different tradition of the specific distribution they're targeting.
+
+### Minimum package set
+
+* `dotnet-runtime-[major].[minor]`: a shared framework with the specified version (only the latest patch version for a given major+minor combination should be available in the package manager). **dependencies**: `dotnet-host`
+* `dotnet-sdk`: the latest SDK. **dependencies**: the latest `dotnet-sdk-[major].[minor]`.
+* `dotnet-sdk-[major].[minor]`: the SDK with the specified version. The version specified is the highest included version of included shared frameworks, so that users can easily relate an SDK to a shared framework. **dependencies**: `dotnet-host`, one or more `dotnet-runtime-[major].[minor]` (one of those is used by the SDK code itself, the others are here for users to build and run against).
+* `dotnet-host`: the latest host.
+
+#### Preview versions
+
+Package maintainers may decide to include preview versions of the shared framework and SDK. Those should never be included in the unversioned `dotnet-sdk` package, but may be released as versioned packages with an additional preview marker appended to the major and minor version sections of the name. For example, there may be a `dotnet-sdk-2.0-preview-final` package.
+
+### Optional additional packages
+
+Some maintainers may choose to provide additional packages such as:
+
+* `dotnet-lts`: the latest "Long-Term Support" version of the shared framework. [LTS and Current release trains](https://docs.microsoft.com/en-us/dotnet/articles/core/versions/lts-current) correspond to different cadences at which .NET Core gets released. Users have a choice of adopting one or the other train, based on how often they are willing to update. This is a concept that is also tied to support levels, so it may or may not make sense depending on the distro being considered.
+
+## Disk layout
+
+When installing .NET Core packages, the relative placement of their target destinations on disk matter.
+The `dotnet.exe` host should be placed next to `sdk` and `shared` folders that contain the versioned contents of the `dotnet-sdk` SDK packages, and `dotnet-runtime` shared framework packages.
+
+The disk layout of files and directories inside the packages is versioned. This means that updating to the latest `dotnet-runtime` will effectively install the new version side-by-side with the ones that were previously there, reducing the possibility of breaking existing applications by updating the package. Package updates should not remove previous versions.
```

Comment:
will effectively install -> installs
side-by-side should not be hyphenated here. only hyphenated when used as an adjective.
with the ones that were previously there -> with previous versions